### DISCUSSION TITLE
dhcp on 9.0.1

### DISCUSSION THREAD

**Discussion by awolff64** [coreemu/core#720] [Q&A] [answered]
Am getting deeper into it ... (great tool!!!)

I want to configure a dhcp server on a router.
I did not find this service under _Config Services_
However I did find it under _Services (Deprecated)_ and seems to work.

What is the proper way to set up a dhcp server on an interface?

Thanks for your help!

Andreas

**Comment by bharnden** [comment, marked as answer]
The config service counterparts exist, although they do check to validate that the expected commands to use them are found (`dhcpd`, `dhclient`).

These are just baked in example services, they are not meant to necessarily cover all desires or possibilities. You can run whatever commands you like to achieve your desired case, or script that out, or implement a [custom service](https://coreemu.github.io/core/configservices.html#new-services) to tailor things to your exact needs.

So there is no proper way in all reality to do anything, just meeting the needs of whatever software you are using.
